ext/drbbdup/drbbdup.c

```diff
@@ -1389,6 +1412,7 @@ drbbdup_instrument_dups(void *drcontext, void *tag, instrlist_t *bb, instr_t *in
     } else if (drbbdup_is_at_end(instr)) {
         /* Handle last special instruction (if present). */
         if (is_last_special) {
+            // NOCHECK
```

Comment:
What does NOCHECK mean here, please? I am only familiar with this term in an SQL context.

Reply:
My mistake: that is my marker for something that needs attention and it should have been removed after adding the handling.  Removed.